helpers/cache.js

```diff
@@ -46,10 +46,8 @@ module.exports.connect = function(cacheEnabled, config, logger, cb) {
 
 	client.on('ready', () => {
 		logger.info('App connected with redis server');
+		isRedisLoaded = true;
 
-		if (!isRedisLoaded) {
-			isRedisLoaded = true;
-		}
 		return cb(null, { cacheEnabled, client });
```

Comment:
I think this callback was inside the if block as well. See: https://github.com/LiskHQ/lisk/commit/efa364cdf84e187a0ba75193b269059bdf10eb86#diff-664899618f3e4fae8b73a2bb455fc7ffR53 

I can see that isRedisLoadedCheck is redundant. But just wanted to confirm, are you sure we don't need that if check.

Reply:
Yes, I think it's not a problem.

Reply:
Client fires the `ready` event if it loses the connection with Redis and afterward connects again. Therefore, the `cb` should be called inside the `if` condition, otherwise it can be called multiple times, which will kill the application.